install/kubernetes/cilium/templates/cilium-envoy/daemonset.yaml

```diff
@@ -60,6 +60,10 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.envoy.extraInitContainers }}
+      initContainers:
+      {{- toYaml .Values.envoy.extraInitContainers | nindent 6 }}
+      {{- end }}
```

Comment:
Sorry I'm not super familiar with the templating syntax. Why not using the ` with .Values.envoy.extraInitContainers ... toYaml . |` syntax, as for the other items? And why a different level of indentation? Is this because `extraInitContainers` is an array?

Reply:
agreed, let's be consistent with the directives above.